presentation/src/main/java/com/anytypeio/anytype/presentation/navigation/AppNavigation.kt

```diff
@@ -36,7 +36,7 @@ interface AppNavigation {
 
     fun exit()
     fun exitToDesktop()
-    fun openPageSearch(initialQuery: String, space: Id)
+    fun openPageSearch(space: Id)
```

Comment:
let's rename this legacy function to `openGlobalSearch`

Reply:
done